landing-page.html

```diff
@@ -56,7 +56,7 @@ <h2>Support The Coding Train</h2>
   <div class="support">
     {% capture patreonLogo %}{% include 4-vector-graphics/patreon.svg %}{% endcapture %}
     {% include 3-modules/hero-item.html image=patreonLogo title='Consider supporting?' content='Join the Coding Train
-    community with a YouTube membership or Patreon subscription!' url=site.links.patreon cta='Become a Patron' class='patreon' %}
+    community with a YouTube membership or Patreon subscription!' url=site.links.patreon cta='Become a Member' class='patreon' %}
```

Comment:
We need to change the url to point to the youtube page or wherever the join button takes you on that page. Can define new links [in _config.yml](https://github.com/CodingTrain/website/blob/43164955539617a09bc7fcc30334873eb635ea84/_config.yml#L47)